arches_lingo/views/api/pythonic_models.py

```diff
@@ -43,6 +45,17 @@ class SchemeNamespaceView(ArchesModelAPIMixin, RetrieveUpdateDestroyAPIView):
     serializer_class = SchemeNamespaceSerializer
 
 
+class SchemeCreationView(ArchesModelAPIMixin, RetrieveUpdateDestroyAPIView):
+    permission_classes = [RDMAdministrator]
+    serializer_class = SchemeCreationSerializer
+
+
+class TextualWorkRdmSystemSerializerView(ArchesModelAPIMixin, ListCreateAPIView):
```

Comment:
If we don't need to create these, we can reduce to ListAPIView, but if that's follow up work then no sweat.

Reply:
Thanks for the pointer.